src/version/features/UserTour/styles.js

```diff
@@ -1,4 +1,6 @@
-import { MAIN_THEME_COLOR } from "../../config/theme.config";
+import { getCurrentThemeColor } from "../../../core/config/styles";
+
+const themeColor = getCurrentThemeColor();
```

Comment:
Again, I don't understand why you don't export a mui theme instead

Reply:
Done